ComfortZoneApp/app/src/main/java/com/example/comfortzone/WeatherDao.java

```diff
@@ -14,10 +14,12 @@ public interface WeatherDao {
     @Query("SELECT * FROM WeatherData")
     List<WeatherData> getAll();
 
-    // WHERE `temp` BETWEEN :max AND :min
-//    @Query("SELECT * FROM WeatherData WHERE `temp` > :temp")
-//    List<WeatherDao> findByTemp(int temp);
-
     @Insert
     void insertAll(WeatherData... weatherData);
+
+    @Query("DELETE FROM WeatherData")
+    void deleteEntireTable();
+
+    @Query("SELECT * FROM WeatherData WHERE timeUploaded < :hourAgo")
```

Comment:
I'm confused, according to your logic above, shouldn't this be `timeUploaded > :hourAgo` since hour ago should be less than timeUploaded for weather data to qualify?